sqlc-gen-ftl/proto/codegen.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto3";
```

Comment:
Is this proto not versioned with the version of sqlc? If it is, it would be better to download the same version as the sqlc version.

Reply:
updated with a script for this